README.md

```diff
@@ -150,6 +150,32 @@ Two methods are currently supported:
 
 Both discovery methods can be used at the same time (`--source parity_txqueue,jsonrpc_getblock`).
 
+### Running with `docker-compose`
+
+1. Create Kovan account with Metamask. This address will be `$ETH_FROM`
+
+2. Create `keystore.json` and `password.txt` with MyCrypto
```

Comment:
we can remove `with MyCrypto` part, there are many ways to do that

Reply:
Are there better alternatives if so what are they? Can list a number of choices. Just trying to make it easy because I didn't know how to get it for example and had to spend some time.

Reply:
there are (you can use script or geth command) but I don't think is the purpose of this readme